UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json

```diff
@@ -0,0 +1,1799 @@
+                  "valueCoding": {
+                    "code": "mm[Hg]",
+                    "system": "http://unitsofmeasure.org"
+                  }
+                }
+              ],
+              "code": [
+                {
+                  "code": "8480-6",
+                  "system": "http://loinc.org"
+                }
+              ],
+              "text": "BP sys (mm[Hg])",
+              "type": "decimal"
+            },
+            {
+              "linkId": "OBJ.1d.2",
+              "extension": [
+                {
+                  "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-unit",
+                  "valueCoding": {
+                    "code": "mm[Hg]",
+                    "system": "http://unitsofmeasure.org"
+                  }
+                }
+              ],
+              "code": [
+                {
+                  "code": "8462-4",
+                  "system": "http://loinc.org",
+                  "display": "BP dias"
+                }
+              ],
+              "text": "BP dias (mm[Hg])",
+              "type": "decimal"
+            },
+            {
+              "linkId": "OBJ.1e",
+              "code": [
+                {
+                  "code": "8302-2",
+                  "system": "http://loinc.org"
+                }
+              ],
+              "text": "Height",
+              "type": "string"
+            },
+            {
+              "linkId": "OBJ.1f",
+              "code": [
+                {
+                  "code": "3141-9",
+                  "system": "http://loinc.org"
+                }
+              ],
+              "text": "Weight",
+              "type": "string"
+            }
+          ]
+        },
+        {
+          "linkId": "OBJ.2",
+          "text": "General Appearance",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.3",
+          "text": "Had and Neck",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.4",
+          "text": "Chest/lungs",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.5",
+          "text": "Cardiovascular",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.6",
+          "text": "Abdominal",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.7",
+          "text": "Musculoskeletal/extremities (including gait exam)",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.8",
+          "text": "Neurological",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.9",
+          "text": "Psychiatric",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.10",
+          "text": "Visual",
+          "type": "text"
+        },
+        {
+          "linkId": "OBJ.11",
+          "text": "Other",
+          "type": "text"
+        }
+      ]
+    },
```

Comment:
Do we need this section and it is  in the template?
![Screen Shot 2020-05-06 at 9 44 53 AM](https://user-images.githubusercontent.com/58572522/81184553-85b36800-8f7e-11ea-9f48-64018fa31f70.png)

Reply:
Oh, I see, you move it together with the treatment plan section.